examples/.gitignore

```
@@ -1,2 +1,2 @@
-simple_trixi_controller_c
-simple_trixi_controller_f
+trixi_controller_simple_c
+trixi_controller_simple_f
```

Should we add
```suggestion
trixi_controller_simple_f
trixi_controller_mpi_c
trixi_controller_mpi_f
trixi_controller_data_c
trixi_controller_data_f
trixi_controller_t8code_c
trixi_controller_t8code_f
```
as well?

---

Yes, thanks!